manager/src/main/resources/application.properties

```diff
@@ -13,9 +13,10 @@ quarkus.smallrye-openapi.info-title=Red Hat Openshift SmartEvents Fleet Manager
 quarkus.smallrye-openapi.info-version=0.0.1
 quarkus.smallrye-openapi.info-description=The API exposed by the fleet manager of the SmartEvents service.
 quarkus.smallrye-openapi.info-license-name=Apache 2.0
-quarkus.smallrye-openapi.info-license-url=https://www.apache.org/licenses/LICENSE-2.0.html
+quarkus.smallrye-openapi.info-license-url=https://www.apache.org/licenses/LICENSE-2.0
 quarkus.smallrye-openapi.info-contact-email=openbridge-dev@redhat.com
 quarkus.smallrye-openapi.info-contact-name=Development Team
+quarkus.smallrye-openapi.servers=https://api.openshift.com,https://api.stage.openshift.com,http://localhost:8000,/
```

Comment:
Should this really include the trailing comma (`http://localhost:8000,`)?

Reply:
it is not trailing comma, it is comma before last server url, if I remove `/` form the list, I will get:
```
   14:3   warning  rhoas-servers-config           OpenAPI `servers` object is missing the following required URLs: /                              servers[0]
```

Reply:
Thank-you for pointing it out :smile: